firestore/integration_test_internal/src/settings_test.cc

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Google LLC
+
+#include <future>
+#include <map>
+#include <string>
+
+#include "firebase/firestore.h"
+#include "firestore_integration_test.h"
+
+namespace firebase {
+namespace firestore {
+
+using SettingsTest = FirestoreIntegrationTest;
+
+TEST_F(SettingsTest, Equality) {
+  int64_t five_mb = 5 * 1024 * 1024;
```

Comment:
Optional: make these `constexpr` and name them like constants (`kFiveMb`)?

Reply:
Nit: please include `stdint.h`.

Reply:
Done